anker/src/state.rs

```diff
@@ -32,6 +34,14 @@ pub struct Anker {
     #[serde(serialize_with = "serialize_b58")]
     pub b_sol_mint: Pubkey,
 
+    /// Token swap data. Used to swap stSOL for UST.
+    #[serde(serialize_with = "serialize_b58")]
+    pub pool: Pubkey,
```

Comment:
Nit: maybe call it `token_swap_pool` or something? “pool” is rather generic, it might be confused for the reserve.